### PR TITLE
add UI, firebase services & auth functionality

### PR DESCRIPTION
**Pull request by PuneetP16**
Added:
- magnificent CSS
- firebase config
- basic home page 
- firebase authentication services
- Auth pages and Functionality

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,5 +1,3 @@
-@import url("https://magnificentui.netlify.app/src/magnificent.css");
-
 .body {
 	background-color: var(--COLOR-BG);
```
Comment (dev-ashishsethi) [inline src/App.css:2]: give fallback values for variables

File: src/App.css [hunk 2]
```diff
@@ -24,11 +22,11 @@
...
 		"aside main"
 		"footer footer";
-} */
+}
 
-/* .auth {
+.auth {
 	grid-template-columns: 1fr;
-	grid-template-rows: 5.5rem 1fr 5.2rem;
+	grid-template-rows: 7rem 1fr 5.2rem;
```
Comment (dev-ashishsethi) [inline src/App.css:29]: use fractions in all the 3 sizes or rems in all

File: src/App.js [hunk 3]
```diff
@@ -1,9 +1,78 @@
...
 function App() {
-	return <div className="App">
-    Home Page
-  </div>;
+	const { pathname } = useLocation();
+	const { alert } = useAlert();
+	const { authState } = useAuth();
+	const { loader } = useLoader();
+
+	const isAuth = !!authState?.token;
```
Comment (dev-ashishsethi) [inline src/App.js:24]: didin't know about "!!" thanks

**Review by dev-ashishsethi — APPROVED** [review]
Do check comments

**Review by m26unkwn — APPROVED** [review]
Your code quality great
1. Like the BEM convention for css
2. Naming of variable is good
3. Like that you are using Firebase

**Pull request merged**